test/specs/manifestBuilder.e2e.ts

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { step } from 'mocha-steps';
+import path from 'path';
+import { TextEditor } from 'wdio-vscode-service';
+import { TestSetup } from '../testSetup';
+import * as utilities from '../utilities';
+
+describe('Manifest Builder', async () => {
+  let testSetup: TestSetup;
+  const fiveMinutes = 5 * 60;
+
+  step('Set up the testing environment', async () => {
+    utilities.log(`ManifestBuilder - Set up the testing environment`);
+    testSetup = new TestSetup('ManifestBuilder', false);
+    await testSetup.setUp();
+
+    utilities.log(
+      `${testSetup.testSuiteSuffixName} - calling createCustomObjects()`
+    );
+    utilities.createCustomObjects(testSetup);
+
+    utilities.log(`${testSetup.testSuiteSuffixName} - creating manifest file`);
+
+    const workbench = await browser.getWorkbench();
+
+    // Using the Command palette, run File: New File...
+    const inputBox = await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'Create: New File...',
+      1
+    );
+
+    // Set the name of the new manifest file
+    const filePath = path.join('manifest', 'manifest.xml');
+    await inputBox.setText(filePath);
+
+    // The following 3 confirms are just confirming the file creation and the folder it will belong to
+    await inputBox.confirm();
+    await inputBox.confirm();
+    await inputBox.confirm();
+    const editorView = workbench.getEditorView();
+    const textEditor = (await editorView.openEditor(
+      'manifest.xml'
+    )) as TextEditor;
+    const content = [
+      `<?xml version="1.0" encoding="UTF-8"?>`,
+      `<Package xmlns="http://soap.sforce.com/2006/04/metadata">`,
+      `\t<types>`,
+      `\t\t<members>Customer__c</members>`,
+      `\t\t<members>Product__c</members>`,
+      `\t\t<name>CustomObject</name>`,
+      `\t</types>`,
+      `\t<version>57.0</version>`,
+      `</Package>`
+    ].join('\n');
+
+    await textEditor.setText(content);
+    await textEditor.save();
+    await utilities.pause(1);
+    utilities.log(
+      `${testSetup.testSuiteSuffixName} - finished creating manifest file`
+    );
+  });
+
+  // TODO: Select components and magically generate manifest file without the context menu :clown_face:
```

Comment:
I think what might be better is:
1. remove the block of commented code (lines 70-110)
2. move the code which manually creates the manifest file into its own step, and add comments.  Here is the code for the first two steps:
```
  step('Set up the testing environment', async () => {
    utilities.log(`ManifestBuilder - Set up the testing environment`);
    testSetup = new TestSetup('ManifestBuilder', false);
    await testSetup.setUp();
  });

  step('Generate Manifest File', async () => {
    // Normally we would want to run the 'SFDX: Generate Manifest File' command here, but it is only
    // accessible via a context menu, and wdio-vscode-service isn't able to interact with
    // context menus, so instead the manifest file is manually created:
    utilities.log(
      `${testSetup.testSuiteSuffixName} - calling createCustomObjects()`
    );
    utilities.createCustomObjects(testSetup);

    utilities.log(`${testSetup.testSuiteSuffixName} - creating manifest file`);

    const workbench = await browser.getWorkbench();

    // Using the Command palette, run File: New File...
    const inputBox = await utilities.runCommandFromCommandPrompt(
      workbench,
      'Create: New File...',
      1
    );

    // Set the name of the new manifest file.
    const filePath = path.join('/manifest/manifest.xml');
    await inputBox.setText(filePath);

    // The following 3 confirms are just confirming the file creation and the folder it will belong to.
    await inputBox.confirm();
    await inputBox.confirm();
    await inputBox.confirm();

    const editorView = workbench.getEditorView();
    const textEditor = (await editorView.openEditor(
      'manifest.xml'
    )) as TextEditor;
    const content = [
      `<?xml version="1.0" encoding="UTF-8"?>`,
      `<Package xmlns="http://soap.sforce.com/2006/04/metadata">`,
      `\t<types>`,
      `\t\t<members>Customer__c</members>`,
      `\t\t<members>Product__c</members>`,
      `\t\t<name>CustomObject</name>`,
      `\t</types>`,
      `\t<version>57.0</version>`,
      `</Package>`
    ].join('\n');

    await textEditor.setText(content);
    await textEditor.save();
    await utilities.pause(1);
    utilities.log(
      `${testSetup.testSuiteSuffixName} - finished creating manifest file`
    );
  });

  step('SFDX: Deploy Source in Manifest to Org', async () => {
    .
    .
    .
```

What are you thoughts on this?

Reply:
Yeah, I think it looks better and more organized :) I only had all the commented part like a placeholder in case we eventually get access to the context menu, but we'll see then